### ISSUE TITLE
not working with RN 0.22.2?

### ISSUE THREAD

**Issue by matejkriz** [Qwikly/react-native-router-redux#36]
anybody make this working latest React Native? (0.22.2)

actions.routes.myRoute() simply do nothing for me :-/

**Comment by matejkriz** [comment]
oh, actions.routes.myRoute()() actually change the route...

**Comment by jbrodriguez** [comment]
Right, I'm not sure why the route functions are written like thunk functions.

@malkomalko Can you change them to regular action creator functions ?
